test/Spring-test.js

```diff
@@ -167,26 +258,64 @@ describe('Spring', () => {
       },
     });
     TestUtils.renderIntoDocument(<App />);
+
+    // Initial render
+    expect(count).toEqual([0]);
+    mockRaf.manySteps(8);
+    expect(count).toEqual([
+      0,
```

Comment:
nit: `round` would actually make regression pass if previously it was 0.7 and now it's 0.8. Never gonna happen but still, no need to round I think. Plus, yes, it's deterministically wrong if ever, so we do want that.